_pages/welcome-to-TTS/classes/gsa-internal-tools.md

```diff
@@ -119,17 +119,17 @@ If you need to reset your ENT password, you can let your password expire and the
 
 ## <a id="meeting-space">Meeting Space</a>
 
-<a href="https://meet.gsa.gov/">Meeting Space</a> is the service you'll use to create your own conference call line. Here&rsquo;s how to do that:
+<a href="https://meet.gsa.gov/">Meeting Space</a> is the service you'll use to create your own conference call line, as opposed to our usual video calls in Google Hangouts Meet. Hangouts Meet allow up to 99 participants plus a host: Meeting Space is usually only needed if you'll go beyond that or your participants can't use Hangouts Meet. Here&rsquo;s how to create a Meeting Space call line:
```

Comment:
Maybe add a link to the [Google Meet page](https://handbook.18f.gov/google-meet/) here?

Reply:
Good call! Also made a few more minor changes if you have a minute.